README.md

```diff
@@ -72,6 +72,28 @@ If you're not using precompiled libs, build the projects in *__BUILD_BEFORE* fol
 `Build > Build Solution`
 
 
+## Building on Windows (MinGW):
+
+1)Install packages
+- `pacman -S base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-yasm mingw-w64-x86_64-python2 mingw-w64-x86_64-ntldd-git mingw-w64-x86_64-qt5 mingw-w64-x86_64-openal mingw-w64-x86_64-glew git`
+2) Clone repository
+- `git clone https://github.com/RPCS3/rpcs3.git` --recursive-submodules
+3) Configure and compile rpcs3
+- `mkdir rpcs3_build && cd rpcs3_build`
+- `export CFLAGS="-O2 -march=native -D__STDC_FORMAT_MACROS=1"`
+- `cmake -G "MSYS Makefiles" -DCMAKE_MAKE_PROGRAM=mingw32-make -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_FLAGS_RELEASE="$CFLAGS" -DCMAKE_CXX_FLAGS_RELEASE="$CFLAGS" ../rpcs3/`
+- `mingw32-make.exe GitVersion && mingw32-make.exe discord-rpc && mingw32-make.exe ffmpeg-mingw && mingw32-make.exe`
```

Comment:
You should just need to run `mingw32-make.exe` ?

Reply:
Running mingw32-make with -jX key may cause race conditions.